test/build/base-url-homepage/base-url-homepage.test.js

```diff
@@ -1,26 +1,40 @@
 const path = require('path');
 const cheerio = require('cheerio');
-const {setupBuildTest, readFiles} = require('../../test-utils');
+const snowpack = require('../../../snowpack');
 
-const cwd = path.join(__dirname, 'build');
+const TEST_ROOT = __dirname;
+const TEST_OUT = path.join(__dirname, 'build');
+let result;
 
-let files = {};
+function getFile(id) {
```

Comment:
We should move this into `test-utils` so that we don't need to define it individually in every test file. Maybe `getBuildResult` instead since `getFile` feels a bit "Node.js fs library"

Reply:
Would this be okay?

```javascript
getBuildResult(result, TEST_OUT, './index.html');
```

The function has to take in the `result` object and the `TEST_OUT` param as they might be different for each test. Or maybe I should curry the function to take the first 2 params on beforeAll?